### DISCUSSION TITLE
Models de dades

### DISCUSSION THREAD

**Discussion by FherStk** [ElPuig/IMS#17] [2. Equip de treball: assistència i fitxa d'alumne]
Es planteja el següent esquema de models de dades:
- **attendance_group**: nosaltres definim els alumnes que haurien d'anar a cada grup de forma predeterminada.
- **attendance_template**: el professor ho crea usant un attendance_group com a plantilla, però és una còpia de dades independent. S'hi afegeixen o treuen alumnes, es pot canviar l'aula, es poden canviar els dies de la setmana, les hores, etc.
- **attendance_session**: el profe ha fet servir un attendance_template per dir que ha fet classe en un dia i hora, amb un conjunt d'alumnes. Aquí serà on marquem (attendance_status) si cadascun d'aquests alumnes hi ha assistit o no, si hi ha incidència, etc.
- **attendance_status**: es on es recull si un estudiant ha assistit a classe o no, si se li ha obert una incidència, etc.

**Comment by ChusVazquez** [comment]
Entenc que tindrem un attendance_session per cada sessió/alumne, que a la seva vegada permet un o varis registres d'attendance_status, oi?

**Comment by FherStk** [comment]
Tindríem un attendace_session per cada sessió (quan el profe vol passar llista i crea una nova instància), i això generaria un attendance_status per cada alumne dins aquesta sessió. Aleshores, el profe només ha de marcar si aquesta status és falta, etc.
